src/clients/IndividualClient.java

```diff
@@ -99,4 +87,72 @@ public void setAccount(IndividualClientsAccount account) {
 		this.account = account;
 	}
 
+	@Override
+	public boolean equals(Object obj) {
+		if (!super.equals(obj))
+			return false;
+		if (getClass() != obj.getClass())
+			return false;
+		if (this.hashCode() != obj.hashCode())
+			return false;
```

Comment:
```suggestion
		if ((!super.equals(obj)) || (getClass() != obj.getClass()) || (this.hashCode() != obj.hashCode()))
			return false;
```
Those conditions are short and easy to read so we can make it oneliner.

Reply:
ok